README.md

```diff
@@ -93,6 +90,9 @@ For a full list of contributors refer to the [authors page](./AUTHORS.md)
 Pose questions and help answer them on [Stack
 Overflow](https://stackoverflow.com/questions/tagged/fairlearn) with the tag
 `fairlearn` or on [Gitter](https://gitter.im/fairlearn/community#).
+If you think your question is broadly applicable and may be useful to other
+users you can open a
+[GitHub Discussion](https://github.com/fairlearn/fairlearn/discussions/new).
```

Comment:
do we have a code of conduct? We need to enforce that on the discussions page if we route people there.

Reply:
We have this currently: https://fairlearn.github.io/v0.5.0/about/index.html#code-of-conduct

I agree. I had always assumed that we need to enforce that in issues & discussions. [Not that there was anything to enforce so far, but obviously it makes a ton of sense to be aware of CoC ahead of time.]